docker/.env

```
@@ -156,23 +156,23 @@ SMQ_USERS_DB_SSL_ROOT_CERT=
 
 ### Email utility
-SMQ_EMAIL_HOST=__SMQ_EMAIL_HOST__
-SMQ_EMAIL_PORT=__SMQ_EMAIL_PORT__
-SMQ_EMAIL_USERNAME=__SMQ_EMAIL_USERNAME__
-SMQ_EMAIL_PASSWORD=__SMQ_EMAIL_PASSWORD__
-SMQ_EMAIL_FROM_ADDRESS=__SMQ_EMAIL_FROM_ADDRESS__
-SMQ_EMAIL_FROM_NAME=Cube AI
+SMQ_EMAIL_HOST=smtp.mailtrap.io
+SMQ_EMAIL_PORT=2525
+SMQ_EMAIL_USERNAME=18bf7f70705139
+SMQ_EMAIL_PASSWORD=2b0d302e775b1e
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SMQ_EMAIL_PASSWORD key should go before the SMQ_EMAIL_PORT key

(UnorderedKey)